docs/development.md

```diff
@@ -90,12 +90,21 @@ Terminology we use:
 
 ### Reviewing a pull request
 
-What to look out for when reviewing a pull request:
+#### What to look out for when reviewing a pull request:
 
   - Have tests been added?
   - Does this feature / fix work locally for me? 
   - Am I able to understand the code correctly / have comments been added to the code?
 
+#### Prow best practices
+
+  - Get used to how prow works and its commands. 
+    - Prow uses [OWNERS](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) files to determine who can approve and lgtm a PR.
+    - List of possible bot [commands](https://prow.k8s.io/command-help)
+  - As far as possible, do not `/lgtm` a PR, until it is mergable (we need atleast 2 approving reviews and no request changes left and pre-submit tests should pass) or if you do, put it on hold with `/hold` immediately. Either of them can be cancelled as `/lgtm cancel` or `/hold cancel`. This especially applies to **draft PRs**
+  - Approvers can use `/approve` and `/approve cancel` respectively to give their approval. **Note:** approvers github approval also counts the same as `/approve`
```

Comment:
Can you please add short explanation how are approves (approvers) different from lgtm (reviewers)?

Reply:
Sure

Reply:
Added